packages/cli/src/util/certs/get-cert-by-id.ts

```diff
@@ -4,7 +4,7 @@ import * as ERRORS from '../errors-ts';
 
 export default async function getCertById(client: Client, id: string) {
   try {
-    return await client.fetch<Cert>(`/v5/now/certs/${id}`);
+    return await client.fetch<Cert>(`/v6/now/certs/${id}`);
```

Comment:
Just to be clear `Cert` is the correct type still? It doesn't need to be modified?

Reply:
Yeah @styfle - v5 could either return `{ cert:  {...} }` or `{ ...cert }` 🥴 

v6 is standardized to `{...cert}`